packages/fx-core/src/plugins/resource/frontend/constants.ts

```diff
@@ -97,6 +97,14 @@ export class DependentPluginInfo {
   static readonly LocalTabEndpoint = "localTabEndpoint";
 }
 
+export class ArmOutput {
+  static readonly simpleAuthEndpoint = "simpleAuth_endpoint";
+  static readonly frontendEndpoint = "frontendHosting_endpoint";
+  static readonly frontendDomain = "frontendHosting_domain";
+  static readonly frontendStorageName = "frontendHosting_storageName";
+  static readonly frontendWebAppName = "frontendHosting_webAppName";
```

Comment:
What's this? I think current frontend hosting solution doesn't introduce web app? Is it for simple auth?

Reply:
the frontendHosting_webAppName is removed. Thanks.